packages/core/app-layout/src/components/AppLayout.cy.ts

```diff
@@ -195,6 +195,18 @@ describe('<AppLayout />', () => {
         cy.get('[data-testid="app-layout-sidebar-top-slot-content"]').should('be.visible')
       })
 
+      it('should render content passed in through sidebar-footer slot', () => {
+        cy.mount(AppLayout, {
+          slots: {
+            'sidebar-footer': [
+              '<span data-testid="app-layout-sidebar-footer-slot-content">Sidebar footer</span>',
+            ],
+          },
+        })
+
+        cy.get('[data-testid="app-layout-sidebar-footer-slot-content"]').should('be.visible')
```

Comment:
Could we instead check for the text content to be `Sidebar footer`?

Reply:
I'm not sure it makes a difference? I'm just following the pattern in the rest of the test file